lib/similar.js

```diff
@@ -23,7 +23,7 @@ function similar (opts) {
 
     const qs = queryString.stringify({
       id: mergedOpts.appId,
-      hl: mergedOpts.lang,
+      hl: 'en',
```

Comment:
why is this hard coded to 'en' now?

Reply:
hrm I just saw @mmoksh comment. Are you positive that you don't get different results from different countries here? perhaps there's something different than the "Similar apps"/"Similar games" text that can be checked so it's not language dependant>?

Reply:
@facundoolano I checked many apps with different countries/languages (including de, jp, ar), couldn't find any case where this fix won't work.

Reply:
@facundoolano If we use a different language here, the `Similar apps/games` text would appear in a different language, and we cannot match the cluster title with the hard-coded text in English.

Now using `'en'` here would guarantee that we will get the correct cluster. The cluster url for different languages is different, but it returns the same list of apps. When the language parameter is appended to the cluster url in the next steps, it will get the apps' information in the desired language.

Reply:
Example: https://play.google.com/store/apps/details?id=com.matchingham.impossible.date.tricky.riddles&hl=de

The cluster url for German is: [Link](https://play.google.com/store/apps/collection/cluster?gsr=SlNqGC9VNkJLS2U4QjEwY3NnNUxMWDNtdVE9PcICNgoyCi5jb20ubWF0Y2hpbmdoYW0uaW1wb3NzaWJsZS5kYXRlLnRyaWNreS5yaWRkbGVzEAcYCA%3D%3D:S:ANO1ljI2_Dk)

The cluster url for English is: [Link](https://play.google.com/store/apps/collection/cluster?gsr=SlNqGHpmVFY4NXJvQy80d0ZnZDFNeFh1akE9PcICNgoyCi5jb20ubWF0Y2hpbmdoYW0uaW1wb3NzaWJsZS5kYXRlLnRyaWNreS5yaWRkbGVzEAcYCA%3D%3D:S:ANO1ljLuO20)

When appending `hl=de` to the English cluster, it returns the same apps' info the first url gets: [Link](https://play.google.com/store/apps/collection/cluster?gsr=SlNqGHpmVFY4NXJvQy80d0ZnZDFNeFh1akE9PcICNgoyCi5jb20ubWF0Y2hpbmdoYW0uaW1wb3NzaWJsZS5kYXRlLnRyaWNreS5yaWRkbGVzEAcYCA%3D%3D:S:ANO1ljLuO20&hl=de)

Reply:
@facundoolano I tried to find a different way to match the similar apps cluster, but couldn't find any. I even checked all properties for different clusters and couldn't find a different except the apps list, the title, and the random values that change with each request.

Reply:
@facundoolano  Let me know how you want to handle this.

Reply:
We can leave it like this